### ISSUE TITLE
ERROR --> E0078: RPC method did not return 200 OK: ERROR: Unable to connect to

### ISSUE THREAD

**Issue by TheCryptoServices** [MPOS/php-mpos#2764]
I'm having this issue with 2 new wallets using  0.16.xx (Digibyte and LiteCoinCash for example)
Everything else working fine, but no way to make auto payments work and I must do manual payments.
I checked all rpc info (user, password, port) and it's all good. Otherwise nothing else will work, 

Any tip to fix that issue? anyone else?

**Comment by cornac** [comment]
Check  Admin panel- Funds - Wallet Info. 
If you see blank screen then check Apache error logs when you click this menu. Accounts method is deprecated in latest versions of daemons. Add   “deprecatedrpc=accounts” to coin config

**Comment by TheCryptoServices** [comment]
Hi!
Thanks for answer.
Already know that issue and actually using deprecatedrpc=accounts in my conf file.
Wallets page and everything else working fine, but still same error with payments.

It seems payment cron cannot connect with rpc wallet:

ERROR --> E0078: RPC method did not return 200 OK: Address: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
ERROR: Unable to connect to http://xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx@192.168.5.50:20026/

But everything else working fine, so has nonsense it cannot connect with rpc...

**Comment by cornac** [comment]
may be you use encrypted wallet and you need to unlock it to payout?

**Comment by TheCryptoServices** [comment]
No.. no encrypted wallets, the only thing I see different is the 3 seconds wait to "confirm" the payment.
May that be the cause? Can that be the cause?

**Comment by TheSerapher** [comment]
When connecting the Bitcoin RPC call does a getmininginfo call to check if it was able to connect properly. That call does no return a 200 - maybe the implementation changed? You can go through the code and check which calls are done and find those that are not working properly anymore.

If you find a generic way to fix it without breaking other coins feel free to create a PR!
